spec.html

```diff
@@ -464,14 +578,18 @@ <h1>ExecuteModule ( <ins>_capability_</ins>)</h1>
         1. Assert: _module_ has been linked and declarations in its module environment have been instantiated.
         1. Set the VariableEnvironment of _moduleCxt_ to _module_.[[Environment]].
         1. Set the LexicalEnvironment of _moduleCxt_ to _module_.[[Environment]].
-        1. <del>Suspend the currently running execution context.</del>
-        1. <del>Push _moduleCxt_ on to the execution context stack; _moduleCxt_ is now the running execution context.</del>
-        1. <del>Let _result_ be the result of evaluating _module_.[[ECMAScriptCode]].</del>
-        1. <del>Suspend _moduleCxt_ and remove it from the execution context stack.</del>
-        1. <del>Resume the context that is now on the top of the execution context stack as the running execution context.</del>
-        1. <del>Return Completion(_result_).</del>
-        1. <ins>Perform ! AsyncBlockStart(_capability_, _module_.[[ECMAScriptCode]], _moduleCxt_).</ins>
-        1. <ins>Return.</ins>
+        1. Suspend the currently running execution context.
+        1. <ins>If _module_.[[ModuleAsync]] is *false*, then</ins>
+          1. <ins>Assert: _capability_ was not provided.</ins>
+          1. Push _moduleCxt_ on to the execution context stack; _moduleCxt_ is now the running execution context.
+          1. Let _result_ be the result of evaluating _module_.[[ECMAScriptCode]].
+          1. Suspend _moduleCxt_ and remove it from the execution context stack.
+          1. Resume the context that is now on the top of the execution context stack as the running execution context.
+          1. Return Completion(_result_).
+        1. <ins>Otherwise,</ins>
+          1. <ins>Assert: _capability_ was provided.</ins>
+          1. <ins>Perform ! AsyncModuleStart(_capability_, _module_.[[ECMAScriptCode]], _moduleCxt_, _module_).</ins>
```

Comment:
I can't see where the `AsyncModuleStart` is that is being referenced here? Should this have stayed `AsyncBlockStart`?

Reply:
You're right, this should be AsyncBlockStart.